src/Components/test/testassets/BasicTestApp/BasicTestApp.csproj

```diff
@@ -11,7 +11,6 @@
 
     <!-- Project supports more than one language -->
     <BlazorWebAssemblyLoadAllGlobalizationData>true</BlazorWebAssemblyLoadAllGlobalizationData>
-    <UseRazorSourceGenerator>false</UseRazorSourceGenerator>
```

Comment:
With local and CI-based runs of `./build.sh` in `src/Components` I'm not seeing https://github.com/dotnet/aspnetcore/issues/31023 anymore so I think we're good here.

If that turns out not to be the case, we can revisit.